tests/analyse/test_scenario.py

```diff
@@ -846,6 +847,62 @@ def test_model_modelled_obs_ch4(model_scenario_ch4_dummy, footprint_dummy, flux_
     modelled_mf = combined_dataset["mf_mod"].values
     assert np.allclose(modelled_mf, expected_modelled_mf)
 
+@pytest.fixture
+def obs_dummy():
+    """
+    Create example ObsData object with dummy data
+     - Species is methane (ch4)
+     - Hourly frequency for 2012-01-01 - 2012-01-02 (48 time points)
+     - "mf" values are from 1, 48
+    """
+    from openghg.dataobjects import ObsData
```

Comment:
Move this import to the start of the file

Reply:
done